### PR TITLE
change native python IO

### PR DESCRIPTION
**Pull request by sunxiaobiu**
Since I can't register Google cloud platform successfully with my credit card, there is no available gs:// path to use.

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -62,8 +63,7 @@ def build_vocab(self, sentences):
         return [vocabulary, vocabulary_inv]
 
     def preprocess(self, input_file, vocab_file, tensor_file):
-        with open(input_file, "r") as f:
-            data = f.read()
+        data = file_io.read_file_to_string('gs://your-bucket-path/input_file')
```
Comment (hunkim) [inline utils.py]: Why still static? just input_file?

**Comment by hunkim** [comment]
Did you test? There are other file_io APIs. Please check.

**Comment by hunkim** [comment]
At least you need to test it locally.

**Comment by sunxiaobiu** [comment]
I have commit the code that using boto to upload/download file to/from google_cloud and I have tested it by running "python train.py" locally, it works~
btw，you need to prepare the environment first, some param like 'client_id'、'client_secret' in the local {HOME}/.boto file need to be setted:
```
client_id = 857458742632-lrt61p69cvb8bc4sa5lv3k8adterfc3t.apps.googleusercontent.com
client_secret = f58f6QWrjj1qqTZjIrFR5Fna
```
also, in the code , the params need to be settled if you want to use your own google cloud bucket:
```
self.data_dir = data_dir
		self.GOOGLE_STORAGE = 'gs'
		self.TENSORFLOW_BUCKET = 'rnn-tensorflow'
		self.CLIENT_ID = '857458742632-lrt61p69cvb8bc4sa5lv3k8adterfc3t.apps.googleusercontent.com'
		self.CLIENT_SECRET = 'f58f6QWrjj1qqTZjIrFR5Fna'
		self.project_id = 'crucial-raceway-148802'
```

**Pull request closed without merging**